tool-plugins/nextjs-starter/src/components/Test.tsx

```diff
@@ -0,0 +1,27 @@
+import { APP_BRIDGE_TOKEN_HEADER_KEY, KEY_TOKEN } from '@/utils/const';
+import { useEffect, useState } from 'react';
+
+export default function Test() {
+	const [testInfo, setTestInfo] = useState<{ verified: boolean }>({
+		verified: false,
+	});
+	useEffect(() => {
```

Comment:
I would suggest renaming this to something like `Verification.tsx` so that in case the user skips the readme they are not perplexed that this is not a test. The same for the api endpoint either `api/verify` or `api/authorize`? WDYT?

Reply:
Hmmm yeah I'll rename it but not to something `Verification.tsx` or `api/verify`. It's because... unlike Nuxt, we don't have middleware here, and it means users have to implement sending the token from the component, and verifying the token at the corresponding APIs. And I mean everywhere they interact with their APIs. So general names like those can confuse them.

So I think just renaming them to `Example.tsx` and `/api/example` might make more sense.

Reply:
I understand. Yes example makes more sense.